miio/gateway/gateway.py

```diff
@@ -141,6 +133,15 @@ def model(self):
             self._info = self.info()
         return self._info.model
 
+    @property
+    def subdevice_model_map(self):
+        """Return the subdevice model map."""
+        # Check if catch already has the subdevice_model_map, otherwise read it.
```

Comment:
No need for comments that are obvious from the context :-)

Reply:
Remove that